docs/suse/deploy.md

```diff
@@ -0,0 +1,150 @@
+
+```
+kubernetes:
+  authEndpoint: $KUBE_SERVER_ADDRESS
+prometheus:
+  kubeStateMetrics:    
+    enabled: true
+nginx:
+  username: $USERNAME
+  password: $PASSWORD 
+```
+
+Where:
+
+- `$KUBE_SERVER_ADDRESS` is the same URL that you used when registering your Kubernetes environment with Stratos (the Kubernetes API Server URL).
+- `$USERNAME` should be chosen by you as the username that you will use when connecting to Stratos Metrics
+- `$PASSWORD` should be chosen by you as the password that you will use when connecting to Stratos Metrics
+
+> Note: If you omit the `nginx` section in this file, the default username and password will be `metrics` and `s3cr3t`.
+
+Install Metrics with:
+
+```
+helm install suse/metrics \
+    --name susecf-metrics \
+    --namespace metrics \
+    --values scf-config-values.yaml \
+    --values stratos-metrics-values.yaml \
+```
+
+Wait until all pods are in the Ready state.
+
+You can locate the IP and Port that Stratos Metrics is running on with:
+
+```
+kubectl get service susecf-metrics-metrics-nginx --namespace=metrics
+```
+
+This will give output similar to:
+
+```
+NAME                           TYPE       CLUSTER-IP       EXTERNAL-IP   PORT(S)         AGE
+susecf-metrics-metrics-nginx   NodePort   172.24.218.219   10.17.3.1     443:31173/TCP   13s
+```
+
+> Note: If you are using a private container registry, you will need to provide your registry details and credentials in a configuration file - see [here](https://github.com/SUSE/stratos-metrics/blob/master/README.md#deploying-metrics-from-a-private-image-repository) for more details.
+
+## Connecting Stratos Metrics to Stratos
+
+When Stratos Metrics is connected to Stratos, additional views are enabled that show metrics metadata that has been ingested into the Stratos Metrics Prometheus server.
+
+To enable this, you must register and connect your Stratos Metrics instance with Stratos.
+
+In the Stratos UI, go to `Endpoints` in the left-hand side navigation and click on the `+` icon in the top-right on the view - you should be shown the `Register new Endpoint` view.
+
+1. Select `Metrics` from the `Endpoint Type` dropdown
+1. Enter a memorable name for yor environment in the `Name` field
+1. Enter the URL of your Metrics endpoint
+1. Check the `Skip SSL validation for the endpoint` if using self-signed certificates
+1. Click `Finish`
+
+The view will refresh to show the new endpoint in the disconnected state.
+
+Next you will need to connect to this endpoint.
+
+1. In the table of endpoints, click the three-dot menu icon alongside the endpoint that you added above
+1. Click on 'Connect' in the dropdown menu
+1. Enter the username and password for your Stratos Metrics instance
+1. Click `Connect`
+
+Once connected, you should see that the name of your Metrices endpoint is a hyperlink and clicking on it should show basic metadata about the Stratos Metrics endpoint.
```

Comment:
It it worth giving some examples of metrics for user to prove out it's getting data (cf app + pod metrics)?